src/routes/Profile/MyGoals/MyGoals.tsx

```diff
@@ -36,12 +38,16 @@ const AwardIcon = ({ color }: { color: 'gold' | 'silver' }) => {
 };
 
 const MyGoals = () => {
-  const { t } = useTranslation();
+  const {
+    t,
+    i18n: { language },
+  } = useTranslation();
   const routes = useOutletContext<RoutesNavigationListProps['routes']>();
   const navigationRoutes = React.useMemo(() => mapNavigationRoutes(routes), [routes]);
   const title = t('profile.my-goals.title');
   const [addModalOpen, setAddModalOpen] = React.useState(false);
   const revalidator = useRevalidator();
+  const suosikit = useSuosikitStore((state) => state.suosikit);
```

Comment:
Maybe this could be shortened to be:
```tsx
  const suosikitIsEmpty = useSuosikitStore((state) => state.suosikit).length === 0;
```

Then it is more readable and clear from the code when using and with no need then to have same length check two times:

```tsx
//...
  {suosikitIsEmpty && (
//...
//...
  disabled={suosikitIsEmpty}
//...
```